src/functions/runtime.ts

```diff
@@ -0,0 +1,75 @@
+// @ts-nocheck
```

Comment:
Shouldn't be hard to add typings?

Reply:
Yeah actually I don’t remember why I used this